src/plugins/multimedia/multimedia.js

```diff
@@ -680,19 +681,47 @@ $document.on( initializedEvent, selector, function( event ) {
 
 			// finally lets load safely
 			return Modernizr.load( {
-				load: "https://www.youtube.com/iframe_api"
+				load: "https://www.youtube.com/iframe_api",
+
+				//possible solution for multimedia and doaction conflict in corporate network
+				complete: function() {
+					setTimeout( function() {
+						var resources = window.performance.getEntriesByType( "resource" );
```

Comment:
Declare variable at the beginning of the function.

```suggestion
						var resources, arrIframesYt;

						resources = window.performance.getEntriesByType( "resource" );
```

Reply:
Applied. Thanks